static/search-traces.html

```diff
@@ -47,7 +50,7 @@
 
         <div id="app-side-nav" class="active">
         </div>
-
+        <div id="download-trace"></div>
         <div id="dashboard" class="p-20">
```

Comment:
If you remove the bottom padding, the scrolling issue with the navbar will be resolved.
make it - class="p-20 pb-0".